### PR TITLE
[DBIO-2078] refactor: take out get-dbio-pre-register

### PR DESCRIPTION
**Pull request by azmimuhammad**
### JIRA LINK 🟦
- https://blocksphere2020.atlassian.net/browse/DBIO-2078

### Changelog/Description 🗒️
- take out get-dbio-pre-register

### WARNING ⚠️
-

### CODE REVIEW

File: src/store/index.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import testRequest from './testRequest'
 import substrate from './substrate'
 import lab from './lab'
 import feedback from './feedback'
-import registration from './registration'
+// import registration from './registration'
```
Comment (agustinustheo) [inline src/store/index.js:10]: Why is this commented, not deleted?

File: src/views/Dashboard/Lab/NavigationLayout.vue [hunk 2]
```diff
@@ -75,8 +75,8 @@ import ErrorConnectionDialog from "@/components/Dialog/DialogErrorConnection"
 import HeaderNotification from "@/components/HeaderNotification";
 import WalletBinding from "@/components/WalletBinding";
 import DialogAlert from "@/components/Dialog/DialogAlert";
-import { queryBalance } from "@/lib/polkadotProvider/query/balance";
-import localStorage from "@/lib/local-storage";
+// import { queryBalance } from "@/lib/polkadotProvider/query/balance";
+// import localStorage from "@/lib/local-storage";
```
Comment (agustinustheo) [inline src/views/Dashboard/Lab/NavigationLayout.vue:79]: Why is this commented, not deleted?

**Review by agustinustheo — CHANGES_REQUESTED** [review]

File: src/views/Dashboard/Lab/NavigationLayout.vue (hunk 2 above)
Comment (azmimuhammad) [reply src/views/Dashboard/Lab/NavigationLayout.vue]: sorry, i forgot when i testing it

Comment (azmimuhammad) [reply src/views/Dashboard/Lab/NavigationLayout.vue]: updated, thanks a lot @agustinustheo

**Review by agustinustheo — APPROVED** [review]

**Pull request merged**